LibNFS.xs

```diff
@@ -1232,8 +1229,8 @@ set (SV* self_sv, ...)
 
                 continue;
             }
-#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
 
+#ifdef NLNFS_NFS_SET_READDIR_MAX_BUFFER_SIZE
             if (!strcmp(param, "readdir_buffer")) {
```

Comment:
I guess `READDIR_BUFFER_SETTING` should be used here.

Reply:
Done.